src/Stripe.net/Entities/PaymentIntents/PaymentIntentSourceAction.cs

```diff
@@ -0,0 +1,20 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+    using Stripe.Infrastructure;
+
+    public enum PaymentIntentSourceActionType
+    {
+        AuthorizeWithUrl,
+        None,
+        Unknown,
```

Comment:
Can we have `Unknown` as the first value in the enum, so that when you initialize a new `PaymentIntentSourceActionType`, the `Type` field defaults to `Unknown`?

Reply:
Hmmm sure but that breaks the "alphabetical order" rule :)

Reply:
That's fine. I don't think it makes sense for enums anyway -- if we add a new value to an existing enum, we should append it after all existing values, otherwise it's technically a breaking change.